stuff/template/form-template.html

```diff
@@ -232,20 +274,23 @@
 			onkeyup="category_value_changed();"
 			onfocus="this.selectionStart = this.selectionEnd = this.value.length;"
 			autofocus>
-	   <!-- very crude edit icon -->
-           {{if eq .ShowEditToggle true}}
-           <img height="32" id="edit-toggle" style="float:right;"
-                onclick="enable_form(!form_is_enabled);" alt="edit toggle"/>
-           {{end}}
-           </td>
+      <!-- very crude edit icon -->
+            {{if eq .ShowEditToggle true}}
+            <img height="32" class="edit-toggle" style="float:right;"
+                 onclick="enable_form(!form_is_enabled);" alt="toggle edit"/>
+            {{end}}
+      </td>
 	  </tr>
 
 	  <tr>
 	    <td align="right"><label for="fprint">Footprint</label></td>
-	    <td><input type="text" name="footprint" size="10" id="fprint" value="{{.Footprint}}">
+	    <td>
+        <input type="text" name="footprint" size="10" id="fprint"
+         value="{{.Footprint}}">
```

Comment:
Sounds good. but I was finding it really hard to follow the multiple element super long lines so is it ok to keep those line breaks?